### ISSUE TITLE
Please help me understand how resourcegroup works here

### ISSUE THREAD

**Issue by xmubeta** [lyft/presto-gateway#165]
Hi all,

I understood resource groups in Presto. But after I reviewed the code of presto-gateway, I can only see the CURD API for resource group and selector. I didn't find any clue about how it works in presto-gateway. So I am confused. Anyone can give me a hint? Thanks.

**Comment by xmubeta** [comment]
I may find the answer after reading the doc of Trino - https://trino.io/docs/current/admin/resource-groups.html.  The Presto cluster can use MySQL DB as configuration instead of file.  It is not mentioned in Presto doc - https://prestodb.io/docs/current/admin/resource-groups.html.

Please help confirm if my understanding is right. Thanks.

**Comment by endoplasmicR** [comment]
yes, the trino resource groups support dynamic loading when supplied in DB, presto-gateway supports that with:
https://github.com/lyft/presto-gateway#resource-groups-api

**Comment by endoplasmicR** [comment]
also, feel free to join the slack channel for faster responses:
https://trinodb.slack.com/archives/CG9K9MX1V
